sopel/config/core_section.py

```diff
@@ -712,24 +991,51 @@ def homedir(self):
     """The number of seconds acceptable between pings before timing out.
 
     :default: ``120``
+
+    You can increase the timout (for example up to 200s) like this:
+
+    .. code-block:: ini
+
+        timeout = 200
```

Comment:
```suggestion
    You can change the timeout like this:

    .. code-block:: ini

        # increase to 200 seconds
        timeout = 200
```

Not necessarily the best alternative, but I don't like the parenthetical here, and I don't like implying that the timeout can be only increased.

Reply:
Good point!